src/style.css

```diff
@@ -1 +1,44 @@
-/* add styles here */
+.page {
+  margin: 0;
+  padding: 0;
+}
+
+.page__body {
+  margin: 0;
+}
```

Comment:
According to the checklist, avoid using tag names for styling elements. It's recommended to style elements using classes. The `.page` and `.page__body` classes seem to be meant for the `<body>` tag, which is an exception to this rule. However, if `.page` is not the `<body>` tag, consider using a class to style it.